lib/controller.rb

```diff
@@ -75,6 +75,36 @@ class SlowFood < Sinatra::Base
      erb :menu
   end
 
+  get '/owner' do
+    @starter = Dish.all(category: 'starter')
+    @main_course = Dish.all(category: 'main_course')
+    @dessert = Dish.all(category: 'dessert')
+    erb :owner
+  end
+
+  # def test_hello_form_post
+  #     post '/new_dish/', params={:name, :price :category}
+  #
+  #   end
+  # end
+
+  post '/owner' do
+      ownername = params[:name]
+      ownerprice = params[:price]
+      ownercategory = params[:category]
+      owner2 = Dish.create(:name => ownername, :price => ownerprice, :category => ownercategory)
```

Comment:
We want dishes to be *associated* with a restaurant owner. That is, a dish `belongs_to` a `Restaurant`. This is in the Datamapper documentation. When you get that set up on the `Dish` and `Restaurant` models, you will have access to things like `.dishes` and `.restaurant`. You will also be able to make that association this way: `Dish.create(restaurant: current_user, price: ownerprice, category: ownercategory)`